<launch>
  <arg name="node_name" default="swipe_dish_planner_ros"/>
  <node name="$(arg node_name)" pkg="swipe_across_the_dishes" type="ros_node.py" output="screen" >
    <rosparam command="load" file="$(find swipe_across_the_dishes)/config/config.yaml" />
  </node>
</launch>